pom.xml

```diff
@@ -4,20 +4,61 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.balconyseats</groupId>
-    <artifactId>vertx-application-support</artifactId>
+    <groupId>io.github.balcony-seats</groupId>
+    <artifactId>vertx-helper</artifactId>
     <version>0.9.0</version>
+    <packaging>jar</packaging>
+
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Vertx helper library</description>
+    <url>https://github.com/balcony-seats/vertx-application-support</url>
```

Comment:
Project repository name should also be changed to artifact id value 'vertx-helper'.

Reply:
Renamed